### PR TITLE
Dev 0.2.0 to merge to main

### PR DESCRIPTION
**Pull request by jlurien**
- Merge dev-0.2.0 to main with currently approved PRs
- Rename main version to 0.2.0-wip

### CODE REVIEW

File: code/API_definitions/location.yaml [hunk 1]
```diff
@@ -8,7 +8,7 @@ info:
   license:
     name: Apache 2.0
     url: https://www.apache.org/licenses/LICENSE-2.0.html
-  version: 0.1.0
+  version: 0.2.0
```
Comment (jlurien) [inline code/API_definitions/location.yaml:11]:
```suggestion
  version: 0.2.0-wip
```

**Review by bigludo7 — COMMENTED** [review]
Add Partial, matchRate and fixed example.

**Comment by jlurien** [comment]
Fine, I didn't update doc by now. There is an issue in Commonalities to simplify the doc template and avoid duplication with yaml, we can make further adjustments when next release is ready.

**Comment by jlurien** [comment]
@bigludo7 Is there anything else to be changed before merging?

**Review by bigludo7 — APPROVED** [review]
Look good for me

**Pull request merged**